Code/Framework/AzToolsFramework/Tests/EditorTransformComponentSelectionTests.cpp

```diff
@@ -596,25 +618,18 @@ namespace UnitTest
     {
         AzToolsFramework::ed_viewportStickySelect = true;
```

Comment:
Seems like every test in EditorTransformComponentSelectionViewportPickingManipulatorTestFixture starts with the same 3 lines - could they be added to the fixture setup?

Reply:
Good call-out, so there are two sides to this... 🙂

The first is (after some consideration) that these lines are actually intentionally here to make reading the test clearer in each case.

I'm about to add a set of tests for when 'sticky select' is false, and I want every test to be explicit about whether it's testing one or the other, having this here it's immediately clear. 

With the camera and entity positions, this is part of the setup for each test, and while I could in theory move it to the fixture, having the calls here makes it explicit in the 'Given' part of the test (it is actually more hidden by putting it in the fixture's `SetUp` call).

The practical reason for not doing this is because currently as we use Mixins for the fixture and test framework part, it makes having a SetUp call for both tricky 🙁 This is something we will likely need to revisit in future to provide an alternative, but I didn't want to get bogged down with that refactor/update right now... it is on my radar though.

Does that seem reasonable to you?

Reply:
Sure, that all makes sense. Just wanted to raise it as a suggestion.

Reply:
Cool thanks, yup definitely good to call-out, I had gone back and forward on it a bit